src/main/ruby/truffleruby/core/refinement.rb

```diff
@@ -21,8 +21,12 @@ def import_methods(*modules)
       if mod.ancestors.length > 1
         warn("#{mod} has ancestors, but Refinement#import_methods doesn't import their methods", uplevel: 1)
       end
-      Primitive.refinement_import_methods(self, mod)
+      Primitive.import_methods(self, mod)
     end
     self
   end
+
+  def refined_class
+    Primitive.refined_class(self)
+  end
```

Comment:
**suggestion:** TBH I don't see any reason to define the method in Ruby (and then call a primitive) in comparison to defining a `@CoreMethod` in Java. Is this primitive used (or will be used) in some other places?

Reply:
Indeed, it should be a `@CoreMethod` now that there is RefinementNodes